packages/@aws-cdk/tmp-toolkit-helpers/src/api/hotswap/common.ts

```diff
@@ -26,7 +26,7 @@ export interface HotswapOperation {
   /**
    * Applies the hotswap operation
    */
-  readonly apply: (sdk: SDK) => Promise<void>;
+  readonly apply: (sdk: SDK, timeoutSeconds?: number) => Promise<void>;
```

Comment:
Should we predict that there will be more options in the future, and move to an `Options` struct straight away?

Reply:
+1 (also this not public currently so maybe not super important)

Reply:
Added a` HotswapOperationOptions` struct.